msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -20,7 +20,16 @@ Your bot receives messages from Teams using the `Text` property and it sends sin
 
 For more information, see [user attribution for bot messages](/microsoftteams/platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit?tabs=dotnet%2Cdotnet-1#user-attribution-for-bots-messages).
 
-## Receive a message
+The following table lists the activity that your bot can receive and take action on:
+
+| Type | Payload object | Scope |
+| ---- | ---------| ------ |
+| [Receive a message activity](#receive-a-message-activity) | Message activity | All |
+| [Receive edit message activity](#get-edit-message-activity) | Message edit activity | All |
+| [Receive undelete message activity](#get-undelete-message-activity) | Message undelete activity | All |
+| [Get soft delete message activity](#get-soft-delete-message-activity) | Message soft delete activity | All |
```

Comment:
Update "Get soft delete message activity" to "Receive soft delete message activity".

Reply:
Updated as suggested